ConsoleApp/Program.cs

```diff
@@ -0,0 +1,196 @@
+}
+(string, string, string)[] ReadContacts(string file)
+{
+    var lines = File.ReadAllLines(file);
+    var contacts = new (string, string, string)[lines.Length];
+    for (int i = 0; i < lines.Length; i++)
+    {
+        var items = lines[i].Split(',');
+        contacts[i] = (items[0], items[1], items[2]);
+    }
+
+    return contacts;
+}
+
+string Serialize((string firstName, string lastName, string phone) row) => $"{row.firstName},{row.lastName},{row.phone}";
+
+void showAllContacts()
+{
+    var lines = ReadContacts(filePath);
+    ShowRow(("First Name", "Last Name", "Phone"));
+    foreach (var contact in lines)
+    {
+        ShowRow(contact);
+    }
+
+    System.Console.WriteLine("Press any key to continue...");
+    Console.ReadKey();
+}
+void exitProg()
+{
+    Environment.Exit(0);
+};
+
+void addNewContacts()
+{
+    Console.Clear();
+    System.Console.Write("Enter first name: ");
+    var firstName = Console.ReadLine();
+    System.Console.Write("Enter second name: ");
+    var lastName = Console.ReadLine();
+    System.Console.Write("Enter phone: ");
+    var phone = Console.ReadLine();
+    File.AppendAllLines(filePath, new[] { Serialize((firstName, lastName, phone)) });
+}
+
+void RemoveContact()
+{
+    Console.Clear();
+
+    System.Console.WriteLine("Enter phone to remove:");
+    var phoneToRemove = Console.ReadLine();
+
+    var contacts = ReadContacts(filePath);
+    var newContacts = new (string, string, string)[contacts.Length];
+    Array.Copy(contacts, newContacts, contacts.Length);
+
+    var index = 0;
+
+    while (index < newContacts.Length)
+    {
+        if (newContacts[index].Item3 == phoneToRemove)
+        {
+            newContacts[index] = newContacts[^1];
+            Array.Resize(ref newContacts, newContacts.Length - 1);
+        }
+        else
+        {
+            ++index;
+        }
+    }
+
+    var csvBuilder = new StringBuilder();
+    foreach (var contact in newContacts)
+    {
+        csvBuilder.AppendLine(Serialize(contact));
+    }
+
+    File.WriteAllText(filePath, csvBuilder.ToString());
+
+    System.Console.WriteLine($"{contacts.Length - newContacts.Length} Contact(s) removed, press any key to continue");
+    Console.ReadKey();
+}
+
+void searchContact()
+{
+    System.Console.WriteLine("Enter Last\\second name or number: ");
+    string searchContact = Console.ReadLine();
+    var contacts = ReadContacts(filePath);
+    var index = 0;
+
+    for (int i = 0; i < contacts.Length; i++)
+    {
+        if (contacts[index].Item1 == searchContact
+        || contacts[index].Item2 == searchContact
+        || contacts[index].Item3 == searchContact)
+        {
+            ShowRow(contacts[i]);
+        }
+    }
+    System.Console.WriteLine("Press any key to continue...");
+    Console.ReadKey();
+
+}
+
+void UpdateContact()
+{
+    var contacts = ReadContacts(filePath);
+    ShowRow(("First Name", "Last Name", "Phone"));
```

Comment:
you've copypasted it from method that shows phone book, so it can be extracted to method